chrysom/client.go

```diff
@@ -281,15 +294,16 @@ func (c *Client) sendRequest(owner, method, url string, body io.Reader) (respons
 }
 
 // GetItems fetches all items that belong to a given owner.
-func (c *Client) GetItems(owner string) (Items, error) {
-	response, err := c.sendRequest(owner, http.MethodGet, fmt.Sprintf("%s/%s", c.storeBaseURL, c.bucket), nil)
+func (c *Client) GetItems(ctx context.Context,owner string) (Items, error) {
+	response, err := c.sendRequest(ctx,owner, http.MethodGet, fmt.Sprintf("%s/%s", c.storeBaseURL, c.bucket), nil)
 	if err != nil {
 		return nil, err
 	}
 
 	if response.Code != http.StatusOK {
+		spanIDHeaderName,spanID,traceIDHeaderName,traceID := extractTraceInformation(ctx,c.headerConfig)
 		level.Error(c.logger).Log(xlog.MessageKey(), "Argus responded with non-200 response for GetItems request",
-			"code", response.Code, "ErrorHeader", response.ArgusErrorHeader)
+			"code", response.Code, "ErrorHeader", response.ArgusErrorHeader,spanIDHeaderName,spanID,traceIDHeaderName,traceID)
```

Comment:
Was `gofmt` run on these files? Normally there are spaces between parameters.

Reply:
I missed it in this PR, will do it next  commit.